docs/rdd-programming-guide.md

```diff
@@ -871,7 +871,7 @@ The most common ones are distributed "shuffle" operations, such as grouping or a
 by a key.
 
 In Java, key-value pairs are represented using the
-[scala.Tuple2](http://www.scala-lang.org/api/{{site.SCALA_VERSION}}/index.html#scala.Tuple2) class
+[scala.Tuple2](https://www.scala-lang.org/api/current/scala/Tuple2.html) class
```

Comment:
shall we still keep `site.SCALA_VERSION`? `https://www.scala-lang.org/api/current/scala` may point to scala 2.13 in the future.

Reply:
That's fine, I found the new URL pattern. I really doubt it will change, but hey.